src/pages/policy.page.js

```diff
@@ -499,22 +503,27 @@ goToCurrentAdaptationPolicy () {
 
   async getTotalNumberOfRecordsOfPolicy() {
     const numberOfRecordsOfPolicy = await I.grabNumberOfVisibleElements(this.table.tableRows);
+    I.say('Rows count is '+numberOfRecordsOfPolicy)
```

Comment:
can use a template literal here

```suggestion
    I.say(`Rows count is ${numberOfRecordsOfPolicy}`)
```